### PR TITLE
[4.0] Correcting #24833 to add the space in the php and not in the lang string

### PR DESCRIPTION
**Pull request by infograf768**
See https://github.com/joomla/joomla-cms/pull/24833#issuecomment-490376846

### After patch
same results as in #24833 without modifying the lang string and adding a comment

### CODE REVIEW

File: plugins/system/skipto/skipto.php [hunk 1]
```diff
@@ -68,7 +68,7 @@ public function onAfterDispatch()
 						'menuLabel'      => Text::_('PLG_SYSTEM_SKIPTO_SKIP_TO_AND_PAGE_OUTLINE'),
 						'landmarksLabel' => Text::_('PLG_SYSTEM_SKIPTO_SKIP_TO'),
 						'headingsLabel'	 => Text::_('PLG_SYSTEM_SKIPTO_PAGE_OUTLINE'),
-						'contentLabel'   => Text::_('PLG_SYSTEM_SKIPTO_CONTENT'),
+						'contentLabel'   => ' ' . Text::_('PLG_SYSTEM_SKIPTO_CONTENT'),
```
Comment (brianteeman) [inline plugins/system/skipto/skipto.php:72]: can you add a comment explaining there is a space etc.

Comment (infograf768) [reply plugins/system/skipto/skipto.php:72]: I would not know how to phrase this as the js is quite obscure. See the other issue I posted where some stuff can’t be translated ( Here the word Main) and RTL was not considered.

Comment (brianteeman) [reply plugins/system/skipto/skipto.php:72]: Just use the string I wrote in the language file

Comment (infograf768) [reply plugins/system/skipto/skipto.php:72]: Done, but see https://github.com/joomla/joomla-cms/issues/24836 as these words make no sense in other languages than English.

Comment (infograf768) [reply plugins/system/skipto/skipto.php:72]: done

**Comment by wilsonge** [comment]
Thanks

**Comment by infograf768** [comment]
### Note
If "Main" is only used to be displayed in the dropdown as "Main Content" (In English), then we can modify the js (Are we allowed to?) to use `val = this.config.contentLabel;` and modify the string to make it multilang aware by using a simple phrase.
`val = this.config.contentLabel; `
In that case the space will be unnecessary.
`PLG_SYSTEM_SKIPTO_CONTENT="Main Content"`

**Comment by wilsonge** [comment]
We're not allowed to modify the JS

**Pull request merged**